Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org' do
-  group :development do
+  gem 'spoon' # Added because jruby seems to want it?
```

Comment:
Addressed in https://github.com/holidays/definitions/pull/1/commits/0dfd6e77f04eed5af2ff93a02c21134240472a60